test/UnburnableToken.t.sol

```diff
@@ -296,9 +297,3 @@ contract UnburnableTokenTest is Test {
         );
     }
 }
-
-library AddressGenerator {
```

Comment:
Extracted to its own file under `test/support` to share it with other test files. Currently used in `UnburnableToken.t.sol` and `HaikuNFT.t.sol`